brainglobe_atlasapi/atlas_generation/atlas_scripts/validation.py

```diff
@@ -0,0 +1,98 @@
+import shutil
```

Comment:
Looks like this file is a local validation script for a different atlas, and probably added to this PR in error?

Reply:
yes apologies, I'll remove it